src/cbor/CBOREncoder.h

```diff
@@ -24,6 +24,7 @@
 
 #ifdef __AVR__
 #include <Arduino.h>
+#include <avr_stl.h>
 #include <ArduinoSTL.h>
```

Comment:
@facchinm If we include `<avr_stl.h>` is there still need to include `<ArduinoSTL.h>` 

CC @per1234